packages/babel-preset/src/transformPlugin.ts

```diff
@@ -41,11 +41,11 @@ function getDefinitionPathFromCallExpression(
 
   const definitionsPath = argumentPaths[0];
 
-  if (!definitionsPath.isObjectExpression()) {
-    throw definitionsPath.buildCodeFrameError(`${functionKind}() function accepts only an object as a param`);
+  if (definitionsPath.isExpression() || definitionsPath.isSpreadElement()) {
```

Comment:
Would be nice to have a check for those cases you don't imagine happening... `JSXNamespacedName | ArgumentPlaceholder`